data/role/jenkins-dev.yaml

```diff
@@ -1,10 +1,7 @@
 ---
-psick::base::linux_classes:
-  openssh: psick::openssh
-
 
 # Enabled and configure scm-sync plugin
-psick::jenkins::scm_sync_repository_url: git@github.com:graduway/jenkins-config-dev.git
+DISABLEpsick::jenkins::scm_sync_repository_url: git@github.com:graduway/jenkins-config-dev.git
```

Comment:
@alvagante Is `DISABLE` valid syntax? Is there a reason why we don't just delete the line completely?

Reply:
@antgel it's a quick way to "comment" the entry, could be any string, or also a normal # .
Anyway can be totally removed, but there are other hiera settings which might be redundant, and actually, now that we have our data separated, we can clean up a lot of sample entries in hieradata module, if you want

Reply:
@alvagante Thanks. Please edit the commit to remove the unused code instead of "commenting" it out, this is a general principle of our workflow. I'll create an Issue for the further cleanup, it's outside scope of this PR.

Reply:
for prod, qa, sandbox leave it as is. Change it when we configure jcasc for these envs.
For dev removed at all.